bubble-wrap.yaml

```diff
@@ -852,7 +852,7 @@ styles:
                 WALL_TINT: vec3(1., 3., .800)
             blocks:
                 color: |
-                    if (dot(vec3(0., 0., 1.), worldNormal()) < 1.0 - TANGRAM_EPSILON) {
+                    if (dot(vec3(0., 0., 1.), worldNormal()) < .5) {
```

Comment:
There are no slanting building tops in tangram. This is the simplest way to distinguish between vertical and horizontal vectors.